umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -6,11 +6,63 @@
   [cmr.umm-spec.date-util :as date-util]
   [cmr.umm-spec.umm-spec-core :as spec-core]))
 
+(def DataCenterURL
```

Comment:
Nit: Clojure style is generally `data-center-url` and `home-page` for constants. I've seen us also use screaming snake case for constants: `DATA_CENTER_URL`. I'm good with either of those, but we probably shouldn't add CamelCase. I think the only place we end up using camel case is from the UMM JSON models.

Reply:
Thanks. Done.

Reply:
Yeah, just a quick additional note on Camel case: it's also used in protocol and record/type names. 

In some open source Clojure projects, constants used in more than one module are put into a `const` ns. We should avoid global vars, but the classic pattern there in Clojure (inherited from Common Lisp) is earmuffs around kebab-case, e.g., `*some-global-const*`, but that's less and less common in the Clojure ecosystem.